src/components/HighlightChat/HighlightChat.tsx

```diff
@@ -78,6 +85,15 @@ const HighlightChat = () => {
   // HOOKS
   useHandleClipboardPaste();
 
+  useEffect(() => {
+    trackEvent('hl_chat_state_changed', { 
+      isChatting,
+      messageCount: messages.length,
+      inputIsDisabled,
+      hasPrompt: !!promptName
+    });
+  }, [isChatting, messages.length, inputIsDisabled, promptName]);
```

Comment:
This will probably fire _a lot_, may not be worth having this event.

Reply:
Removed